modules/swagger-codegen/src/main/resources/Java/apiInvoker.mustache

```diff
@@ -106,12 +106,12 @@ public class ApiInvoker {
 
     Builder builder = client.resource(host + path + querystring).accept("application/json");
     for(String key : headerParams.keySet()) {
-      builder.header(key, headerParams.get(key));
+      builder = builder.header(key, headerParams.get(key));
```

Comment:
I see. However, I found the `header` method from the source code of jersey-client v1.7 downloaded [here](http://download.java.net/maven/2/com/sun/jersey/jersey-client/1.7/):

```java
    public T header(String name, Object value) {
        getMetadata().add(name, value);
        return (T)this;
    }
```

As you can see the method just returns itself, so theoretically speaking, assigning the result again would make no difference, right?